textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexMatcher.java

```diff
@@ -34,9 +34,13 @@
     private List<Span> spans;
 
     public RegexMatcher(IPredicate predicate, ISourceOperator sourceOperator) {
```

Comment:
Alternatively, we can explicit write the `public RegexMather(RegexPredicate predicate, ISourceOperator)`